src/backends/git-gateway/implementation.js

```diff
@@ -98,6 +109,14 @@ export default class GitGateway {
       } else if (this.backendType === "gitlab") {
         this.api = new GitLabAPI(apiConfig);
         this.backend = new GitLabBackend(this.config, { proxied: true, API: this.api });
+      } else if (this.backendType === "bitbucket") {
+	this.api = new BitBucketAPI({
+          ...apiConfig,
+          requestFunction: this.requestFunction,
+          hasWriteAccess: async () => true,
+        });
+        console.log(this.api);
+	this.backend = new BitBucketBackend(this.config, { proxied: true, API: this.api });
```

Comment:
indentation lol. obviously due to your stubborn use of emacs. also we need prettier.

Reply:
:+1: to getting prettier in